src/bidiMapper/modules/browser/ContextConfigStorage.ts

```diff
@@ -23,13 +23,14 @@ export class ContextConfigStorage {
   #browsingContextConfigs = new Map<string, ContextConfig>();
 
   updateGlobalConfig(config: ContextConfig) {
-    this.#global = {...this.#global, ...config};
+    this.#global = {...this.#global, ...this.#removeUndefined(config)};
   }
 
   updateBrowsingContextConfig(
     browsingContextId: string,
     config: ContextConfig,
   ) {
+    config = this.#removeUndefined(config);
```

Comment:
how do we get undefined in the first place? and why does it matter? removal of undefines on a strictly typed data structure does not sound good

Reply:
The `undefined` can be set here: https://github.com/GoogleChromeLabs/chromium-bidi/blob/c66d125b659c704242111570782cb20afb4b3f75/src/bidiMapper/modules/browser/BrowserProcessor.ts#L114-L115

Another approach would be to implement `mergeConfig` method ignoring the `undefined`. WDYT?

Reply:
done